tests/ConfigurationTest.php

```diff
@@ -23,17 +23,6 @@ public function testDoesNotAcceptBadApiKey()
         new Configuration([]);
     }
 
-    public function testDefaultEndpoint()
-    {
-        $this->assertSame($this->config->getNotifyEndpoint(), 'https://notify.bugsnag.com');
```

Comment:
User can set the base uri by calling `$client->getGuzzle()->something else i can't remember`.